extra_geom/tests/test_agipd_geometry.py

```diff
@@ -44,6 +44,25 @@ def check_result(img, centre):
         img, centre = geom.position_modules_fast(stacked_data, threadpool=tpool)
         check_result(img, centre)
 
+
+def test_assemble_symmetric():
```

Comment:
Ideally we should also check that `ValueError` is properly raised too :wink:

Reply:
Thanks Yury, I've added a check at the end of this test.